### PR TITLE
Add pdb

### PR DESCRIPTION
**Pull request by kasturinarra**
Adding poddisruptionbudget

### CODE REVIEW

**Comment by kasturinarra** [comment]
@zhouying7780 could you please help review ? Thanks !!

**Comment by akostadinov** [comment]
`RESOURCE` is missing api name. Could you post full output of `oc get -o yaml` of an example object and I will tell you how it should look.

**Comment by akostadinov** [comment]
Should be
```
RESOURCE = "poddisruptionbudget.policy"
```

**Comment by kasturinarra** [comment]
@akostadinov have updated, could you please help check ? Thanks !!

File: lib/openshift/pod_disruption_budget.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module BushSlicer
+  class PodDisruptionBudget < ClusterResource
```
Comment (akostadinov) [inline lib/openshift/pod_disruption_budget.rb:2]: from output
```
namespace: openshift-kube-descheduler-operator
```

So this object should be `ProjectResource`, not `ClusterResource`.

Comment (kasturinarra) [reply lib/openshift/pod_disruption_budget.rb]: Thought so, but me missed to update as well, thanks !!

**Pull request merged**